tests/integration/test_base.py

```diff
@@ -47,7 +47,7 @@
 
 
 class BaseIntegrationTest(SeleniumBaseTest):
-    default_css_selector = 'section.themed-xblock.xblock--drag-and-drop'
+    default_css_selector = 'div.themed-xblock.xblock--drag-and-drop'
```

Comment:
Note that we prefer to just use class names for selectors instead of including the HTML tag type. It makes changes like you are doing now easier, and the tag type usually doesn't provide any additional value.